tests/multinode.rs

```diff
@@ -89,7 +89,96 @@ fn converge(
         .filter(|x| x.id != me)
         .map(|x| x.clone())
         .collect();
-    v.clone()
+    (v.clone(), me)
+}
+
+#[test]
+fn test_multi_node_validator_catchup_from_zero() {
+    logger::setup();
+    const N: usize = 5;
+    trace!("test_multi_accountant_stub");
+    let leader = TestNode::new();
+    let alice = Mint::new(10_000);
+    let bob_pubkey = KeyPair::new().pubkey();
+    let exit = Arc::new(AtomicBool::new(false));
+
+    let leader_bank = Bank::new(&alice);
+    let server = Server::new_leader(
+        leader_bank,
+        None,
+        leader.data.clone(),
+        leader.sockets.requests,
+        leader.sockets.transaction,
+        leader.sockets.broadcast,
+        leader.sockets.respond,
+        leader.sockets.gossip,
+        exit.clone(),
+        sink(),
```

Comment:
I think we need to sink to tmp file, and let the validators restart from there.

Reply:
This is a confusing comment, since it's on the `sink()` line. When you say "sink to a tmp file", do you mean "sync to a tmp file"?